### PR TITLE
Mention salting and SHA256 block size

### PR DESCRIPTION
**Pull request by jonasnick**
SHA256 block size mention as suggested by @Sjors

### CODE REVIEW

File: bip-taproot.mediawiki [hunk 1]
```diff
@@ -253,6 +253,7 @@ Using fresh keys implies that taproot output construction does not need to take
 This does not avoid leaking information through the leaf depth and therefore only applies to balanced (sub-) trees.
 In addition, every leaf should have a set of keys distinct from every other leaf.
 The reason for this is to increase leaf entropy and prevent an observer from learning an undisclosed script using brute-force search.
+In the unlikely case that a leaf does not contain keys or the keys have to be reused, the tapscript can be salted by adding a push of 16 bytes of random data to the stack, followed by ''OP_DROP''.
```
Comment (sipa) [inline bip-taproot.mediawiki]: I'm not sure about advising this. If someone is willing to go this length in order to get distinct addresses, they could just as well use different keys.

**Comment by sipa** [comment]
ACK the first part

File: bip-taproot.mediawiki (hunk 1 above)
Comment (jonasnick) [reply bip-taproot.mediawiki]: Okay, removed that part.

**Comment by sipa** [comment]
Rebased in #67.

**Pull request closed without merging**